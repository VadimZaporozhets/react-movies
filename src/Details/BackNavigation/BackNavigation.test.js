import React from 'react';
import { BackNavigationComponent } from './index';
import { shallow } from 'enzyme';

describe('<BackNavigationComponent />', () => {
    it('should render back navigation component', () => {
        const props = {
            classes: expect.any(Object)
        };

        const wrapper = shallow(<BackNavigationComponent {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
});
