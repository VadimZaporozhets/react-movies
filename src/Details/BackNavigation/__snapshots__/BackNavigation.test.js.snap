// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackNavigationComponent /> should render back navigation component 1`] = `
<WithStyles(Grid)
  container={true}
  spacing={40}
>
  <WithStyles(Grid)
    item={true}
    xs={12}
  >
    <WithStyles(Paper)
      elevation={1}
    >
      <Link
        replace={false}
        to="/"
      >
        <WithStyles(Button)
          size="medium"
          variant="outlined"
        >
          ‚Üê Back to Home
        </WithStyles(Button)>
      </Link>
    </WithStyles(Paper)>
  </WithStyles(Grid)>
</WithStyles(Grid)>
`;
